vk_generator/src/generator/mod.rs

```diff
@@ -718,9 +731,26 @@ impl<'a> GenTypes<'a> {
                                 }
                                 writeln!(structs, "{},", &*ident)
                             }
-                            MutArray(ident, count)    => writeln!(structs, "{}: [{}; {}],", &*f.field_name, &*ident, count),
-                            MutArrayEnum(ident, cons) => writeln!(structs, "{}: [{}; {}],", &*f.field_name, &*ident, &*cons),
-
+                            MutArray(ident, count) => {
+                                let mut fty = &*ident;
+                                if gen_types.config.remove_type_prefix && fty.starts_with("VK_") {
+                                    fty = &fty[3..];
+                                }
```

Comment:
I don't think a situation would arise where this branch is executed. The `ident` field holds the type contained in the array, which the Vulkan XML styles as CamelCase, so it would never start with `VK_`. Also, prefix stripping isn't performed in this function - it's done in the `GenPreproc` struct.